src/renderer/dx/DxFontInfo.cpp

```diff
@@ -126,44 +126,52 @@ void DxFontInfo::SetFromEngine(const std::wstring_view familyName,
     try
     {
         face = _FindFontFace(localeName);
+    }
+    CATCH_LOG();

```

Comment:
Suppressing whitespace changes makes this a lot easier.
tl;dr: This PR wraps every single `_FindFontFace` call into its own try/catch block and moves the font-variant search code down below the nearby search.